scripts/local-airflow.sh

```diff
@@ -16,5 +17,11 @@ ln -s $PWD/$target_dag $AIRFLOW_HOME/dags/
 
 cd $AIRFLOW_HOME
 
+# Disable auth login
+echo "AUTH_ROLE_PUBLIC = 'Admin'" > webserver_config.py
```

Comment:
Good find! I was wondering how to do this. Can you remove the part of the `contributing` doc that mentions a temporary admin password?